time-utils.c

```diff
@@ -0,0 +1,10 @@
+#include "time-utils.h"
+
+char* get_iso_time(){
```

Comment:
the iso-date length is always the same, therefore I would create and return whole table instead of pointer. The dynamic memory allocation isn't necessary and I would avoid it, whenever we can, because it can lead to memory leaks, that are hard to detect